spec/api-browser-window-spec.ts

```diff
@@ -1544,6 +1544,16 @@ describe('BrowserWindow module', () => {
           await expect(once(w, 'resized')).to.eventually.be.fulfilled();
         });
       });
+
+      it('does not emits the resize event for move-only changes', async () => {
+        const [x, y] = w.getPosition();
+
+        w.once('resize', () => {
+          expect.fail('resize event should not be emitted');
+        });
+
+        w.setBounds({ x: x + 10, y: y + 10 });
```

Comment:
Just note that this doesn't test the `isWithinOnePixel` condition. That one is only used if the user is manually dragging the window title bar with the mouse. It's still a good idea to have this test though!